src/PerfView/Utilities/commandLine.cs

```diff
@@ -1423,7 +1423,11 @@ private object ParseValue(string valueString, Type type, string parameterName)
         {
             try
             {
-                if (type == typeof(string))
+                if (type == typeof(DateTime))
+                {
+                    return DateTime.Parse(valueString);
+                }
+                else if (type == typeof(string))
```

Comment:
Nit: Can you move this further down the list below enum?  DateTime is going to be one of the least common options.